trunk/SUAVE/Analyses/Stability/AVL.py

```diff
@@ -424,9 +433,10 @@ def build_surrogate(self):
         self.surrogates.Cn_beta_moment_coefficient  = cn_beta_surrogate
         self.surrogates.neutral_point               = neutral_point_surrogate
 
-        AoA_points                                  = np.linspace(-3.,11.,100)*Units.deg 
-        mach_points                                 = np.linspace(.02,.9,100)           
-
+        # Standard subsonic/transolic aircarft  
```

Comment:
transonic

Reply:
Issue Resolved

Reply:
You fixed a different one. This still needs to be changed